kolibri/plugins/learn/assets/src/views/ExploreLibrariesPage/index.vue

```diff
@@ -125,7 +159,7 @@
         return this.moreDevices < this.unpinnedDevices?.length;
       },
       networkDevicesWithChannels() {
-        return Object.values(this.networkDevices)
```

Comment:
So, here's the issue - we need to read this from the `useDevices` composable's `networkDevices` otherwise all the work you've done there to ensure that no longer contactable devices get evicted from our map will have gone to waste, because our `deviceChannelsMap` is never evicting anything.

So this should be reverted, the `deviceChannelsMap` should be `[instance_id]: [<channel_1>, <channel_2>....]` and the filter below should be:

```
.filter(device => this.deviceChannelsMap[device.instance_id]?.length > 0)
```